markdown/3.0/en/ff-asn.md

```diff
@@ -1,224 +1,296 @@
 ## Using the After Search Navigation
-You can use the ASN by adding just one line to your HTML code.
+You can use the ASN by adding one line to your HTML code.
 ```html
 <ff-asn></ff-asn>
 ```
-If you do so, the ASN will look pretty much like nothing. So let's style it.
+Doing so will cause the ASN to use default HTML templates for all filters. You can customize the ASN's appearance by providing your own templates for its elements.
```

Comment:
Mh, removing the _example with no effect_ would make it more comprehensive without losing any information, wouldn't it? 

Maybe change the text as followed:

```suggestion
Doing so will cause the ASN to use default HTML templates for all filters. You can customize the ASN's appearance by providing your own templates for its elements. Your customized templates must be wrapped within a `ff-asn-group` element. Below are covered some examples.
```

Reply:
Do you mean `<ff-asn></ff-asn>` by "the example with no effect"? It has the effect that it setups a complete asn with default values/templates.
The custom templates are elaborated on later so I don't see the need to address them here.

Reply:
I meant line 10 ff (as I should have written initial^^)